packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderFinal/index.txt

```diff
@@ -0,0 +1,7 @@
+verificationReminderFinal-description = A couple of weeks ago you created a Firefox account, but never confirmed it. For your security, we will delete the account if not verified in the next 24 hours.
```

Comment:
Forgot the quotes:
verificationReminderFinal-description = "A couple of weeks ago you created a Firefox account, but never confirmed it. For your security, we will delete the account if not verified in the next 24 hours."